src/organisation/components/index.ts

```diff
@@ -0,0 +1,10 @@
+import { PbaNumberInputComponent } from './pba-number-input/pba-number-input.component';
+import { PbaNumbersFormComponent } from './pba-numbers-form/pba-numbers-form.component';
+
+export const components: any[] = [
+    PbaNumberInputComponent,
```

Comment:
Indenting should be 2 spaces rather than 4.

Reply:
done